### PR TITLE
Add feature #7

### PR DESCRIPTION
**Pull request by nyvee**

### CODE REVIEW

File: app/src/main/res/xml/preferences_settings.xml [hunk 1]
```diff
@@ -3,6 +3,16 @@
     <PreferenceCategory
         android:title="@string/settings_account">
 
+        <ListPreference
+            android:key="settings_network"
+            android:title="@string/settings_network"
+            android:summary="@string/settings_network_summary"
+            android:entries="@array/settings_network_entries"
+            android:entryValues="@array/settings_network_entryvalues"
+            android:defaultValue="T"
```
Comment (sunnyking) [inline app/src/main/res/xml/preferences_settings.xml]: Default value maybe should be set to "M"? When the app is first installed, it should be mainnet by default.

File: app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEWallet.java [hunk 2]
```diff
@@ -224,16 +225,18 @@ public VEEWallet(){
         seed = "";
         accountSeeds = new ArrayList<>();
         nonce = 0;
+        chainId = 0;
```
Comment (sunnyking) [inline app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEWallet.java]: Should it be initialized to 'M' here for consistency? As the value is among either 'M' or 'T' above

File: app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEWallet.java [hunk 3]
```diff
@@ -31,7 +32,7 @@
 
     // Choose blockchain ID: MAIN_NET, TEST_NET
     //private static final byte CHAIN_ID = VEEChain.MAIN_NET;
-    private static final byte CHAIN_ID = VEEChain.TEST_NET;
+    // private static final byte CHAIN_ID = VEEChain.TEST_NET;
```
Comment (sunnyking) [inline app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEWallet.java]: If this is no longer needed, should delete the line

File: app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEChain.java [hunk 4]
```diff
@@ -3,4 +3,8 @@
 public class VEEChain {
     public static final byte TEST_NET = 'T';
     public static final byte MAIN_NET = 'M';
+
+    public static byte getChainId(String chainIdString) {
+        return "M".equals(chainIdString) ? VEEChain.MAIN_NET : VEEChain.TEST_NET;
+    }
 }
```
[the remark this answers is not in the document]
Comment (sunnyking) [reply app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEChain.java]: The way this is written, it should be named isMainChain() instead at least

[the remark this answers is not in the document]
Comment (nyvee) [reply app/src/main/java/tech/vee/veecoldwallet/Wallet/VEEChain.java]: I have made some changes

**Pull request merged**